docs/EVE-LOGGING.md

```diff
@@ -35,3 +40,19 @@ When a direct ssh connection to a device is possible, /persist/rsyslog/syslog.tx
 ## Log export to cloud
 
 Rsyslogd is configured to send logs via TCP socket to logmanager. Logmanager bundles the logs into protobuf messages and exports them to cloud using API. Today when a device loses network connectivity to cloud, there is a back-off mechanism built into the logmanager that prompts rsyslogd to stop sending logs. When there is network drop, logmanager stops reading log messages from rsyslogd (TCP socket). Rsyslogd stops sending logs when it's output socket buffer is full. As a result following logs will be queued on disk. After the device's network connectivity is restored, rsyslogd starts sending logs from it's disk queues.
+
+## Log files still present in device
+
+1. Logmanager's log file present in /persist/log/logmanager.log
+   Latest code with cron, logrotate removed from pillar this log file grows irregulated.
+   What can be done to control the total size of this file on disk?
+
+   a. Bring back cron, logrotate?
+   b. Make logmanager send logs directly to /dev/log (not via memlogd) and put a programname
+      match in rsyslogd that redirects these logs into /persist/log/logmanger.log but using a
+	  size controlled out_channel.
```

Comment:
b. makes sense to me. But need to make sure we don't have issues with rsyslogd crashing resulting in a connect to /dev/log failing.

Reply:
I think connect wouldn't be a problem because /dev/log works with datagrams

Reply:
The issue we had before wat that the AF_UNIX connect to the SOCK_DGRAM /dev/log was failing when rsyslog didn't have the socket open (because it hadn't started or had failed).

Reply:
Got it. Will fix it.